### PR TITLE
Settings page documentation updates

### PR DESCRIPTION
**Pull request by suprishi**

### CODE REVIEW

File: documentation/master-devonfw-dashboard.asciidoc [hunk 1]
```diff
@@ -2,24 +2,24 @@
...
 
 == Wiki
 
-include::wiki-page[leveloffset=2]
+include::wiki-page.asciidoc[leveloffset=2]
 
 == Settings
 
-include::settings-page[leveloffset=2]
+include::settings-page.asciidoc[leveloffset=2]
```
Comment (jambulud) [inline documentation/master-devonfw-dashboard.asciidoc]: In include statements you don't have to add .asciidoc

https://raw.githubusercontent.com/devonfw/devon4ng/develop/documentation/master-devon4ng.asciidoc

Comment (suprishi) [reply documentation/master-devonfw-dashboard.asciidoc]: If I remove `.asciidoc`, it shows an error in the preview: Unresolved directive in <stdin> - include::landing-page[leveloffset=2]

Will it work as expected in the Wiki without `.asciidoc`?

Comment (jambulud) [reply documentation/master-devonfw-dashboard.asciidoc]: > If I remove `.asciidoc`, it shows an error in the preview: Unresolved directive in - include::landing-page[leveloffset=2]
> 
> Will it work as expected in the Wiki without `.asciidoc`?

You can check an example of master file in devon4ng wiki: https://github.com/devonfw/devon4ng/wiki/master-devon4ng. In there, it is being shown properly.

Even if it were not being displayed correctly, master-\*.asciidoc are only used by the website generator (for the wiki we already have sidebar.asciidoc) so I prefer it to keep master-devonfw-dashboard.asciidoc like the other master-\*.asciidoc files

Comment (suprishi) [reply documentation/master-devonfw-dashboard.asciidoc]: Corrected.

**Review by jambulud — APPROVED** [review]

**Pull request merged**